# CVE-2026-48109: bump MessagePack extension bundle to 2.5.301 in extensions.json.
# Source: ecosystem/azure-functions/patch/message_pack.patch
# Upstream: https://github.com/Azure/azure-functions-extension-bundles
diff --git a/src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json b/src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json
index da46fe0..71d58e8 100644
--- a/src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json
+++ b/src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json
@@ -277,5 +277,11 @@
       "mcpResourceTrigger",
       "mcpPromptTrigger"
     ]
+  },
+  {
+    "id": "MessagePack",
+    "version": "2.5.301",
+    "name": "MessagePack",
+    "bindings": []
   }
 ]
